app/api_clients/github.js

```diff
@@ -29,6 +32,31 @@ class GithubAPIClient extends GithubAPI {
 
       return Promise.resolve(user.github_access_token);
     };
+
+    async function paginate(method, params = {}) {
+      const defaultParams = { per_page: 100 };
+      const firstPageResponse = await method({ ...defaultParams, ...params });
+      let { data } = firstPageResponse;
+      const lastPage = await this.hasLastPage(firstPageResponse);
+      const requests = [];
+
+      if (lastPage) {
+        // get the last page integer from the full url of the last page
+        const lastPageNum = parseInt(parseQuery(parseUrl(lastPage).query).page, 10);
+
+        // queue up a bunch of requests
+        for (let i = 2; i <= lastPageNum; i += 1) {
+          requests.push(method({ ...defaultParams, page: i, ...params }));
```

Comment:
Since you're merging `...defaultParams` and `...params` for a second time here, it might be cleaner to do it once and assign to a variable.